chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/CertifiedProductDAO.java

```diff
@@ -1,66 +1,671 @@
 package gov.healthit.chpl.dao;
 
+import java.util.ArrayList;
```

Comment:
FYI I have a PR for OCD-3275 ("Reviewed" status) that also removes this Impl. It would be good to make sure we know what other changes were made to the file in each case so we don't lose them because this will be a conflict for sure (I would think if you pushed this to DEV it has already become an issue there?).